pom.xml

```diff
@@ -190,6 +190,11 @@
                     <fail>true</fail>
                     <cvssScoreThreshold>7.0</cvssScoreThreshold>
                     <excludeCoordinates>
+                        <exlude>
```

Comment:
Why have we excluded the vulnerability instead of fixing it? The point of the audit tests is to spot security vulnerabilities so we should be fixing them by upgrading the dependencies rather than just excluding them. If there is still some specific reason why we are excluding them there should be a comment in here explaining why and preferably with a link to a bug ticket for us to come back and fix it properly.

Reply:
This is now addressed.